docs/av-stack-branch.1.md

```diff
@@ -0,0 +1,28 @@
+# av-stack-branch 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-branch - Create or rename a branch in the stack
+
+# SYNOPSIS
+
+`av stack branch [-m | --rename] [--parent <parent_branch>] <branch-name>`
+
+# DESCRIPTION
+
+Create a new branch that is stacked on the current branch by default
+
+If the --rename/-m flag is given, the current branch is renamed to the name
+instead of creating a new branch. Branches should only be renamed
+with this command (not with git branch -m ...) because av needs to update
```

Comment:
Now realizing that we should have a way to retrofit the renames that happen outside of av. People would just do that, and later want to fix that.